Installing-Oppia-using-Docker.md

```diff
@@ -74,6 +74,7 @@ To install Oppia under Docker, follow these steps:
 ### Pre-requisites for setting up the development server
 
 1. **Install Docker Desktop**: Download and install the latest version of Docker Desktop from the [official Docker website](https://www.docker.com/products/docker-desktop/). Docker Desktop provides a user-friendly interface for developers. Follow the steps in the given link to download and install it.
+2. **[For Windows users]**: The Windows users must have `make` installed on their system. To install `make`, follow the steps given in the following link: [Installing make on Windows](https://stackoverflow.com/questions/32127524/how-to-install-and-use-make-in-windows).
```

Comment:
The Windows users --> You
their --> your

Can you explain how a developer can verify they have `make` on their system? The flow here should be "do this to see if you have it; if you don't, follow these instructions".

Reply:
done. PTAL